### ISSUE TITLE
`declaration-block-trailing-semicolon` writes into a `//` comment that a whole node stands in under Less

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#723]
Found by the review of the branch closing [#359](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/359). The branch leaves it as it found it.

### What minimal example or steps are needed to reproduce the bug?

```less
a {
	color: pink // ; top: 0
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "customSyntax": "postcss-less",
  "rules": {
    "@stylistic/less/declaration-block-trailing-semicolon": "always"
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.less" --config myconfig.json --fix
```

### What did you expect to happen?

Nothing reported: Less reads `; top: 0` as the text of the comment behind `color: pink` and compiles `color: pink;` alone, so the block ends on that comment.

### What actually happened?

`--fix` writes `color: pink // ; top: 0;`. The same comment with `top: 0;` under `never` loses its last semicolon, and `color: pink // ; .b { c: d }` under `always` gets `c: d;`. All three are the same on `22af625` and on the branch.

### What is the cause?

`postcss-less` closes `color: pink // ` at the semicolon inside the comment and reads what follows on the line as nodes of their own, so the rule takes `top: 0` for the last declaration of the block. The reading #359 added asks whether the flag of the last node is the comment's text; nothing asks whether the last node itself stands in the comment of the one in front of it.

Measured against `less@4.9.1` and `postcss-less@6.0.0`.
